### DISCUSSION TITLE
Autogen Studio + Ollama models

### DISCUSSION THREAD

**Discussion by romilandc** [microsoft/autogen#2489] [General]
Has Microsoft team announced any plans to streamline Ollama integration into Autogen Studio yet? Seems counterintuitive to me that you build a UI and then have to create scripts to get Ollama models working with that UI.

CrewAI requires similar approach (e.g. https://medium.com/@seliwerhau/using-crewai-for-free-with-llama2-ecfe3f0c3059) however this is because it's not a UI.  Was only able to find this approach with Autogen Studio:  https://gist.github.com/shimomurakei/5692335e4fdfb12450afa2580002db77

Thanks!

**Comment by ekzhu** [comment]
First, we are an open source project, not a product -- we don't announce features. 

You are welcome to submit PR to add the support. See autogen studio roadmap: https://github.com/microsoft/autogen/issues/737

**Comment by romilandc** [comment]
@ekzhu Thanks for the reply Eric- will add to the roadmap.

**Comment by romilandc** [comment]
Just posted to the roadmap, closing this down ty.
